nservicebus/azure-storage-persistence/configuration.md

```diff
@@ -8,31 +8,33 @@ tags:
 - Configuration
 ---
 
-The behavior of each persister can be controlled by specifying one of the respective configuration sections in the `app.config` and changing available properties, or using code API. The exact same settings are exposed via `app.config` file and code configuration API.
+In NServiceBus.Azure the behavior of the AzureStoragePersister can be controlled by specifying one of the respective configuration sections in the `app.config` and changing the available properties, or by using the code API. The exact same settings are exposed via the `app.config` and the code configuration API.
+
+In NServiceBus.Persistence.AzureStorage v1 and above XML based configuration is no longer available. Configuring the behavior of the persister is done using the code configuration API.
 
 ### Configuration with Configuration Section
 
 snippet:AzurePersistenceFromAppConfig
 
 The following settings are available for changing the behavior of saga persistence through the `AzureSagaPersisterConfig` section:
 
-- `ConnectionString`: Sets the connectionstring for the storage account to be used for storing saga information.  Defaults to `UseDevelopmentStorage=true` in Versions 6 and below, and defaults to `null` in Versions 7 and above.
+- `ConnectionString`: Sets the connectionstring for the storage account to be used for storing saga information.  Defaults to `UseDevelopmentStorage=true` in NServiceBus.Azure Versions 6 and below, and defaults to `null` in NServiceBus.Persistence.AzureStorage Version 1 and above.
```

Comment:
Depends on how you define "valid". Is it valid in the sense that it will connect to an Azure storage container? No. The only other option for a default value is to use `UseDevelopmentStorage=true`. The problem there is that many/most people don't use local development storage because it's flakey. I think we even recommend against using it somewhere. That was the primary reason for removing it when we opened the new repo for v1.

If the value is null (or an empty string) the following exception will be thrown:

`System.ArgumentException : ConnectionString should not be an empty string.
Parameter name: connectionString`